guides/common/modules/proc_synchronizing-a-content-view-to-a-smartproxy.adoc

```diff
@@ -0,0 +1,40 @@
+[id="Synchronizing_a_Content_View_to_a_{smart-proxy-context}{context}"]
+= Synchronizing a content view to a {SmartProxy}
+
+You can use the Hammer CLI to synchronize a content view to a {SmartProxy}.
+You need to specify the content view or content view ID along with the lifecycle environment or lifecycle environment ID.
+You will need to include the organization or organization ID as well.
+
+.CLI procedure
+* Synchronize the content view to a {SmartProxy} by using Hammer:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ hammer {hammer-smart-proxy} content synchronize --help
+Usage:
+    hammer capsule content synchronize [OPTIONS]Options:
+ --async                                   Do not wait for the task
+ --content-view[-id] VALUE/NUMBER          Name/id of the content view to limit the synchronization on
+ --environment[-id] VALUE/NUMBER           (-environment-id is deprecated: Use --lifecycle-environment-id instead)
+ --id NUMBER                               Id of the capsule
+ --lifecycle-environment[-id] VALUE/NUMBER Name/id of the environment to limit the synchronization on
+ --name VALUE                              Name to search by
+ --organization VALUE                      Organization name
+ --organization-id VALUE                   Organization ID
+ --repository[-id] VALUE/NUMBER            Name/id of the repository to limit the synchronization on
+ --skip-metadata-check BOOLEAN             Skip metadata check on each repository on the capsule
+ -h, --help                                Print helpOption details:
+  Here you can find option types and the value an option can accept:  BOOLEAN             One of true/false, yes/no, 1/0
+  DATETIME            Date and time in YYYY-MM-DD HH:MM:SS or ISO 8601 format
+  ENUM                Possible values are described in the option's description
+  FILE                Path to a file
+  KEY_VALUE_LIST      Comma-separated list of key=value.
+                      JSON is acceptable and preferred way for such parameters
+  LIST                Comma separated list of values. Values containing comma should be quoted or escaped with backslash.
+                      JSON is acceptable and preferred way for such parameters
+  MULTIENUM           Any combination of possible values described in the option's description
+  NUMBER              Numeric value. Integer
+  SCHEMA              Comma separated list of values defined by a schema.
+                      JSON is acceptable and preferred way for such parameters
+  VALUE               Value described in the option's description. Mostly simple string
+----
```

Comment:
This is not an approapriate way to give instructions. Please, provide a concrete example command that synchronizes a CV and tell users to run `hammer {hammer-smart-proxy} content synchronize --help` under the _Additional resources_ heading below the procedure. Remove the output.

Reply:
Examples:

```
hammer capsule content synchronize --content-view "my content view name"
```

```
hammer capsule content synchronize --content-view-id 42
```

Reply:
I left out some mandatory fields: id and name for the smart proxy. Some better examples:

```
hammer capsule content synchronize --id 1 --organization "Default Organization" --content-view-id 42 
hammer capsule content synchronize --id 1 --organization-id 1 --lifecycle-environment-id 999
hammer capsule content synchronize --name "My Capsule" --organization "Default Organization" --content-view "My CV"
hammer capsule content synchronize --name "My Capsule" --organization-id 1 --lifecycle-environment "Production"
```